fe/src/app/jobs/[jobId]/page.tsx

```diff
@@ -0,0 +1,118 @@
+import Carousel from "@/components/Carousel/Carousel";
+import ChipWithTooltip from "@/components/ChipWithTooltip/ChipWithTooltip";
+import Timer from "@/components/Timer/Timer";
+import { Breadcrumbs, Button, Chip, Typography } from "@/lib/materialTailwindExports";
+import { AdvertType } from "@/types/AdvertType";
+import { ChipTooltipText } from "@/utils/constants";
+import { getToken } from "@/utils/getToken";
+import { BASE_API_URL, SERVICE_ADVERTRS_ROUTE } from "@/utils/urls";
+import Link from "next/link";
+import { FaCheckCircle } from "react-icons/fa";
+import { FaCircleXmark } from "react-icons/fa6";
+
+const AnswerIcon = (isPositive: boolean) => isPositive ? <FaCheckCircle className="text-green-500 text-2xl" /> : <FaCircleXmark className="text-red-500 text-2xl" />;
+
+async function getAdvert(jobId: string) {
+  const token = await getToken();
+  const response = await fetch(`${BASE_API_URL}${SERVICE_ADVERTRS_ROUTE}/${jobId}`, {
+    headers: { Authorization: `Bearer ${token}` }
+  });
+  if (!response.ok) {
+    return;
+  }
+
+  return response.json();
+}
+
+export default async function AdvertInfoPage({ params: { jobId } }: { params: { jobId: string }}) {
+  const advert: AdvertType = await getAdvert(jobId);
+
+  if (!advert) {
+    return "Not found";
+  }
+
+  return (
+    <div className="flex flex-col gap-3 max-w-3xl mx-auto my-5 py-10 px-5">
+      <Breadcrumbs>
+        <a href="/home" className="opacity-60">
```

Comment:
Can Next Js Link be used here?

Reply:
Because if we use a tag, whole app content gets redownloaded to client and Link from "next/link" only fetches the data needed instead of the whole app.